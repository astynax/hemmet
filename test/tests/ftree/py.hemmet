|py|src/*some-package/{*foo/ bar} test/*some-package/{*foo/ bar} README.md !test-cases/!test-case
