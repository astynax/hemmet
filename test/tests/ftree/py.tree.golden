.
├── README.md
├── src/
│   └── some_package/
│       ├── __init__.py
│       ├── bar.py
│       └── foo/
│           └── __init__.py
├── test/
│   └── some_package/
│       ├── __init__.py
│       ├── bar.py
│       └── foo/
│           └── __init__.py
└── test-cases/
    └── test-case
